hello-strongbox-gradle/build.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+compileJava {
+    targetCompatibility = "1.8"
+    sourceCompatibility = "1.8"
+    options.encoding = 'UTF-8'
+}
+
+group = 'org.carlspring.strongbox.examples'
+version = '1.0'
+
+description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
+
+configurations {
+    deployerJars
+}
+
+repositories {
+    maven { url "http://dev.carlspring.org/nexus/content/groups/carlspring" }
```

Comment:
@AlexOreshkevich,

I have tried it without the  first repository, and, as expected, it works, so, please, could you remove this?

Also:
1) The example you have added with the sample output, doesn't display properly. Would you mind fixing this?
2) Could you add an example of a `~/.gradle/gradle.propertie` in the `README.md` for clarity...?

Thanks! This is starting to look quite good!